README.md

```diff
@@ -45,6 +45,54 @@ Yes! This work is still in early development, which means it's _not ready for pr
 
  [![asciicast](https://asciinema.org/a/471709.svg)](https://asciinema.org/a/471709)
 
+## Workspaces in v0.5.0
+
+The workspaces are arranged in a tree.  Each vertex in the tree has a name.  The name of the root vertex is `root`, and no other vetex may have that name.  Other vertices can be addressed by a pathname formed by putting all the path's vertex names together, separated by colons.  For example, the pathname `root:default:foo` identifies the child `foo` of the child `default` of the `root`.
```

Comment:
I like math, but others struggle to follow if we use math terms to explain. Everybody knows hierarchical file system tree. Can we use that to decribe how workspaces relate to each other?